administrator/language/en-GB/com_modules.ini

```diff
@@ -192,6 +192,7 @@ COM_MODULES_TABLE_CAPTION="Table of Modules"
 COM_MODULES_TYPE_CHOOSE="Select a Module Type"
 COM_MODULES_TYPE_OR_SELECT_POSITION="Type or Select a Position"
 COM_MODULES_WORKFLOW="Workflow"
+COM_MODULES_WORKFLOW="Workflows"
```

Comment:
Two language strings with the same name?

Reply:
Oops, sleepy me forgot to plural-ize it. Thanks